src/binding.cc

```diff
@@ -91,4 +91,8 @@ void Init(v8::Local<v8::Object> exports, v8::Local<v8::Value> m_, void* v_) {
       Nan::New(RdKafka::version_str().c_str()).ToLocalChecked());
 }
 
+#if NODE_MAJOR_VERSION >= 10
```

Comment:
AFAIK this directive shouldn't be _needed_ because it's already included in the `NAN_MODULE_WORKER_ENABLED` macro (https://github.com/nodejs/nan/blob/main/nan.h#L165-L178). Did that not work?

Reply:
Thanks a lot! Removed it 🙇